server/controllers/notification.js

```diff
@@ -0,0 +1,97 @@
+const asyncHandler = require("express-async-handler");
+const Notification = require("../models/Notification");
+const User = require("../models/User");
+
+exports.createNotification = asyncHandler(async (req, res) => {
+  const { title, type, senderId, recipientId, message } = req.body;
+  await User.findOne({ _id: senderId }, function (err) {
+    if (err) {
+      res.status(404).send("Sender not found!!");
+    }
+  });
+  await User.findOne({ _id: recipientId }, function (err) {
+    if (err) {
+      res.status(404).send("Receiver not found");
+    }
+  });
+
+  try {
+    const notification = await Notification.create({
+      type,
+      title,
+      recipientId,
+      senderId,
+      message,
+      date,
```

Comment:
Please remove this variable as it has not been declared anywhere :)

Reply:
variable removed